### PR TITLE
V8 migration

### PR DESCRIPTION
**Pull request by mwu2018**
Address #990, #1054

Also includes PR #1051 that addresses #998 

It uses un-published terriajs, which should be updated in the future.

Can be verified by https://nationalmap-2021-09-03.nm.terria.io. In particular, check the following items:
- [x] [Feature info is correct](https://nationalmap-2021-09-03.nm.terria.io/#share=s-atUpOG6cMQaHgiCuTxaEbDYlnw4). Also check if the legends are acceptable - this looks ok; it's the best we can do with the data i.e. once you click on a point, depending on the zoom level, you get the nr of dwellings for corresponding radius + when you check the legend against it, it matches the colour and the type of radius 
- [ ] Is [this group](https://nationalmap-2021-09-03.nm.terria.io/#share=s-iAfRKXpKQjho4IYsOpQGjzFKaAb) needed? It can be removed without code changes - **it can be removed**
- [x] Related maps (#1054)
- [ ] About page - https://github.com/terriajs/nationalmap/issues/1053
- [ ] Privacy page - https://github.com/TerriaJS/nationalmap/issues/1061 - finalised; to update
- [x] Base maps
- [ ] Help page - help doesn't seem updated in https://nationalmap-2021-09-03.nm.terria.io/ but it is in https://beta.nationalmap.terria.io/ 
- [ ] serverconfig. Are there any missing proxied services or credentials?
- [ ] Record any missing features.

### CODE REVIEW

**Comment by mwu2018** [comment]
@AnaBelgun Please check if [this item](https://nationalmap-2021-09-03.nm.terria.io/#share=s-rJQRpw8Hf2kQFlRAsLgtauHh61G) is needed.

File: devserverconfig.json [hunk 1]
```diff
@@ -19,23 +19,11 @@
...
-    ],
-    "initPaths": [
-        "node_modules/nationalmap-catalog/build"
-    ],
-    "feedback": {
-        "userAgent": "TerriaBot (TerriaJS Feedback)",
-        "issuesUrl": "https://api.github.com/repos/TerriaJS/NationalMap-Feedback/issues",
-        "accessToken": null
-    },
-    "maxConversionSize": 20000000
```
Comment (na9da) [inline devserverconfig.json:40]: I think the above chunk is natmap specific config and we have to retain it.

File: index.js [hunk 2]
```diff
@@ -1,44 +1,35 @@
...
+import defined from 'terriajs-cesium/Source/Core/defined';
 
 // Register all types of catalog members in the core TerriaJS.  If you only want to register a subset of them
 // (i.e. to reduce the size of your application if you don't actually use them all), feel free to copy a subset of
 // the code in the registerCatalogMembers function here instead.
-registerCatalogMembers();
-registerAnalytics();
-
-NswTrafficVolume.register();
-NswTrafficHourlyVolume.register();
```
Comment (na9da) [inline index.js:38]: Have to confirm with @AnaBelgun whether we use these catalog items any more? If we do, we'll have to port it to v8.

Comment (na9da) [inline index.js:38]: Have to confirm with @AnaBelgun whether we use these catalog items any more? If we do, we'll have to port it to v8.

File: devserverconfig.json (hunk 1 above)
Comment (mwu2018) [reply devserverconfig.json:40]: This server config is for local development only. The actual config is in S3.
Can add it back though.

File: lib/Views/UserInterface.jsx [hunk 3]
```diff
@@ -18,55 +18,42 @@ import CelebrationStyles from "./Anniversary/celebration.scss";
...
+      </Nav>
+      <ExperimentalMenu>
+        {/* <If condition={isBrowserSupportedAV()}>
+          <SplitPoint
+            loadComponent={loadAugmentedVirtuality}
+            viewState={props.viewState}
+            terria={props.viewState.terria}
+            experimentalWarning={true}
+          />
+        </If> */}
```
Comment (na9da) [inline lib/Views/UserInterface.jsx:54]: I think we should test and if possible enable AR. Also something to check with @AnaBelgun

File: wwwroot/config.json [hunk 4]
```diff
@@ -1,50 +1,166 @@
...
+            ],
+            "title": "Detecting change between two points in time"
+          }
+        ]
+      }
+    ],
+    "mobileDefaultViewerMode": "2d",
+    "regionMappingDefinitionsUrl": "https://raw.githubusercontent.com/TerriaJS/saas-catalogs-public/main/nationalmap/regionMapping-2021-06-02.json",
+    "showWelcomeMessage": true,
+    "supportEmail": "info@terria.io",
```
Comment (na9da) [inline wwwroot/config.json:121]: This email is shown in some error messages. Should we change it to `data@digital.gov.au` from above? /cc @AnaBelgun

**Review by na9da — CHANGES_REQUESTED** [review]
Hi @mwu2018 , looks good to me overall!

I was able to build and run the map without any errors. 

Just have a changes some of which we have to check with @AnaBelgun

File: index.js (hunk 2 above)
Comment (AnaBelgun) [reply index.js:38]: @na9da @mwu2018 Yes, definitely needed in NationalMap. If we are talking about the same data, they are both available in beta already: https://beta.nationalmap.terria.io/#share=s-xPOcwrmfi5ieSnoLYFvhRLpwwY0

File: lib/Views/UserInterface.jsx (hunk 3 above)
Comment (AnaBelgun) [reply lib/Views/UserInterface.jsx:54]: yes, AR should be available in mobile. Are you referring to something else (i.e available in desktop as well?)

File: wwwroot/config.json (hunk 4 above)
Comment (AnaBelgun) [reply wwwroot/config.json:121]: Not at this stage, because NationalMap is being transferred from DTA to GA.

File: index.js (hunk 2 above)
Comment (na9da) [reply index.js:38]: Ok, so just need to make sure those catalog items in beta are available in this repo as well.

Comment (mwu2018) [reply index.js:38]: @AnaBelgun The data you mentioned above are also available here https://nationalmap-2021-09-03.nm.terria.io/#share=s-lqw6o7BUAsE11LVfYneZgeRzvGp

File: lib/Views/UserInterface.jsx (hunk 3 above)
Comment (na9da) [reply lib/Views/UserInterface.jsx:54]: I think it is disabled in terriamap#main too, so we need to enable it there and test.

https://github.com/TerriaJS/TerriaMap/blob/main/lib/Views/UserInterface.jsx#L39-L46

File: index.js (hunk 2 above)
Comment (AnaBelgun) [reply index.js:38]: ok, @mwu2018 as long as it's available, that's ok

**Review by na9da — APPROVED** [review]

**Pull request merged**